string      native_id
---
string      vos_robot_id
